tools/transform.cpp

```diff
@@ -310,6 +310,7 @@ Errors TransformVerify::verify() const {
 
   try {
     sym_exec(src_state);
+    tgt_state.setGlobalVarBidsFrom(src_state);
```

Comment:
maybe call it copyGlobalVarsFromSrc() to make it explicit

Reply:
Updated